### PR TITLE
Support concealed FGI

### PR DESCRIPTION
**Pull request by jrnorth**
Forward port of https://github.com/codice/alliance/pull/901

### CODE REVIEW

**Review by kcwire — APPROVED** [review]

**Comment by jrnorth** [comment]
build now

**Review by samuelechu — APPROVED** [review]

File: catalog/security/banner-marking/src/main/java/org/codice/alliance/security/banner/marking/BannerMarkings.java [hunk 1]
```diff
@@ -27,6 +27,7 @@
 import org.apache.commons.lang.StringUtils;
 
 public class BannerMarkings implements Serializable {
+
```
Comment (shaundmorris) [inline catalog/security/banner-marking/src/main/java/org/codice/alliance/security/banner/marking/BannerMarkings.java:30]: ❔ was this just an extra space added?

**Review by shaundmorris — APPROVED** [review]

File: catalog/security/banner-marking/src/main/java/org/codice/alliance/security/banner/marking/BannerMarkings.java (hunk 1 above)
Comment (jrnorth) [reply catalog/security/banner-marking/src/main/java/org/codice/alliance/security/banner/marking/BannerMarkings.java]: Yeah, though I just realized this wasn't present in the 1.6.x PR so I removed it via a rebase.

**Comment by jrnorth** [comment]
build now

**Pull request merged**